### PR TITLE
Remove policy samples

### PR DESCRIPTION
**Pull request by dockerymick**
https://github.com/open-cluster-management/backlog/issues/8269

### CODE REVIEW

File: security/config_policy_ctrl.adoc [hunk 1]
```diff
@@ -124,9 +124,6 @@ The `exclude` parameter specifies the namespaces you explicitly do not want to a
 `mustnothave`: Indicated that an object with the same name or labels cannot exist and need to be deleted, regardless of the specification or rules.
 |===
 
+See configuration policies samples that are supported by {product-title-short} from the link:https://github.com/open-cluster-management/policy-collection/blob/master/stable/SC-System-and-Communications-Protection/policy-certificate.yaml[CM-Configuration-Management folder]. Learn about how policies are applied on your hub cluster, see xref:../security/policy_sample_intro.adoc#policy-samples[Policy samples] for more details.
```
Comment (dockerymick) [inline security/config_policy_ctrl.adoc]: See the policy examples that use the NIST

**Comment by dockerymick** [comment]
@ycao56 apologies for the long PR, I ran the spellcheck tool and decided to update Rolebinding to role binding. You can ignore the files related to that change. If it is ok with you, I would like to merge this PR and then I will address the comments you have made in the google doc. let me know what you think or prefer

**Review by ycao56 — APPROVED** [review]

File: security/create_rb_policy.adoc [hunk 2]
```diff
@@ -72,30 +72,30 @@ Complete the following steps to create a rolebinding policy from the console:
...
+[#viewing-your-role-binding-policy-from-the-console]
+==== Viewing your role binding policy from the console
 
-You can view any rolebinding policy and its status from the console.
+You can view any role binding policy and its status from the console.
 
 . Log in to your cluster from the console.
 . From the navigation menu, click *Governance and risk* to view a table list of your policies.
 +
 NOTE: You can filter the table list of your policies by selecting the _Policies_ tab or _Cluster violations_ tab.
```
Comment (cadawson) [inline security/create_rb_policy.adoc]: *Note:*

File: security/policy_sample_intro.adoc [hunk 3]
```diff
@@ -1,25 +1,25 @@
...
-* xref:../security/scc_policy.adoc#security-context-constraints-policy[Security context constraints policy sample]
-* xref:../security/cert_policy_ctrl.adoc#certificate-policy-sample[Certificate policy sample]
-* xref:../security/iam_policy_ctrl.adoc#iam-policy-controller[IAM policy sample]
-* xref:../security/gatekeeper_policy.adoc#gatekeeper-policy[Gatekeeper policy sample]
-* xref:../security/etcd_encryption_policy.adoc#etcd-encryption-policy-sample[ETCD encryption policy sample]
+* xref:../security/memory_policy.adoc#memory-usage-policy[Memory usage policy]
+* xref:../security/namespace_policy.adoc#namespace-policy[Namespace policy]
+* xref:../security/pod_nginx_policy.adoc#pod-nginx-policy[Pod nginx policy]
+* xref:../security/psp_policy.adoc#pod-security-policy[Pod security policy]
+* link:../security/role_policy.adoc#role-policy[Role policy]
```
Comment (cadawson) [inline security/policy_sample_intro.adoc]: Will this work? Let me know if it does...

File: security/psp_policy.adoc [hunk 4]
```diff
@@ -99,53 +99,4 @@ Used to list any other Kubernetes object that must be evaluated or applied to th
...
-             supplementalGroups:
-               rule: 'RunAsAny'
-             fsGroup:
-               rule: 'RunAsAny'
-        ...
-----
-
-See xref:../security/create_psp_policy.adoc#managing-pod-security-policies[Managing pod security policies] for more information.
-View other configuration policies that are monitored by controller, see the xref:../security/config_policy_ctrl.adoc#kubernetes-configuration-policy-controller[Kubernetes configuration policy controller] page.
+See link:https://github.com/open-cluster-management/policy-collection/blob/master/stable/SI-System-and-Information-Integrity/policy-psp.yaml[`policy-psp.yaml`] to view the sample policy. View xref:../security/create_psp_policy.adoc#managing-pod-security-policies[Managing pod security policies] for more information. Refer to xref:../security/config_policy_ctrl.adoc#kubernetes-configuration-policy-controller[Kubernetes configuration policy controller] to view other configuration policies that are monitored by controller.
```
Comment (cadawson) [inline security/psp_policy.adoc]: ...monitored by the controller.?

File: security/scc_policy.adoc [hunk 5]
```diff
@@ -108,113 +105,10 @@ For explanations on the contents of a SCC policy, see link:https://docs.openshif
 [#scc-policy-sample]
 == SCC policy sample
 
-Apply an Security context constraints (SCC) policy to control permissions for pods by defining conditions in the policy.
-For more information see, link:https://docs.openshift.com/container-platform/4.3/authentication/managing-security-context-constraints.html#security-context-constraints-about_configuring-internal-oauth[Managing Security Context Constraints (SCC)].
-Your SCC policy might resemble the following YAML file:
+Apply an Security context constraints (SCC) policy to control permissions for pods by defining conditions in the policy. For more information see, link:https://docs.openshift.com/container-platform/4.3/authentication/managing-security-context-constraints.html#security-context-constraints-about_configuring-internal-oauth[Managing Security Context Constraints (SCC)].
```
Comment (cadawson) [inline security/scc_policy.adoc]: Apply a Security context constraints (SCC) policy...

Comment (cadawson) [inline security/scc_policy.adoc]: Can we bump this link up to a later version of OCP? I verified that the same link exists for 4.4, 4.5, and 4.6. All you have to do is change the number in the URL. The reason is that 4.3 is going out of support.

**Review by cadawson — CHANGES_REQUESTED** [review]
Nice job! Looks like the links all work well. I added a few things to look at. After that, it looks good. Thanks!

File: security/policy_sample_intro.adoc (hunk 3 above)
Comment (dockerymick) [reply security/policy_sample_intro.adoc]: I believe it will, i didn't notice it was like that tho. Thanks. I will maintain consistency and change to xref

**Comment by dockerymick** [comment]
@cadawson thank you very much for your review, your feedback was very helpful. I have addressed your comments and will continue to merge this PR. @ycao56 thank you for all of your reviews and time!!

**Pull request merged**